aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -248,7 +248,7 @@ In this section, the **Package Manager Console** (PMC) window is used to:
    ```
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
-* In the **Solution Tool Window**, right-click the *RazorPagesMovie* project, and then select **Open in Terminal**. 
+* Right-click the *RazorPagesMovie* project, and then select **Open in Terminal**. 
```

Comment:
In VSC it's "Open in integrated terminal" but this is probably clear enough.

Reply:
You are right!   I guess last night I read that menu item the way I wanted it to be and not as it was.  I will correct it and make them separate tabs, otherwise it will bother me forever.  Thanks Tom.